android/java/AndroidManifest.xml

```diff
@@ -200,6 +200,7 @@
 
 <service
     android:name="org.chromium.chrome.browser.vpn.wireguard.WireguardService"
+    android:foregroundServiceType="dataSync"
```

Comment:
@samartnik using `dataSync` based on https://developer.android.com/about/versions/14/changes/fgs-types-required#data-sync. Chromium also uses the same for DownloadForegroundService

Reply:
lgtm